setools/terulequery.py

```diff
@@ -141,6 +144,13 @@ def results(self):
         self.log.debug("Boolean: {0.boolean!r}, eq: {0.boolean_equal}, "
                        "regex: {0.boolean_regex}".format(self))
 
+        if self.target and self.target_self:
+            raise ValueError("target and target_self both specified")
+
+        if self.target_self and (self.target_regex or (self.target_indirect is False)):
```

Comment:
Shouldn't this be for either target_regex or target_indirect are true, not false for the latter?

Reply:
Maybe it should just be for target_regex, since neither indirect setting makes sense for self.

Reply:
I believe that target_indirect defaults to True, so checking against (target_regex == True or target_indirect == False) would catch whether either of them are changed from their defaults.  Maybe I'm misunderstanding something about how the current code works?

Reply:
Your understanding is correct. Ok, lets keep the check but it should be more Pythonic:
`if self.target_self and (self.target_regex or not self.target_indirect):`